collector/index.mdx

```diff
@@ -29,3 +29,33 @@ The collector makes only outbound connections, it does not listen on any ports.
 Heroku-based setup, where the collector listens on the dyno's PORT for Heroku to consider the application running,
 as well as to receive database logs by serving as a Heroku log drain.
 
+## Data collection and Snapshot submission Frequency
+
+The pganalyze collector gathers metrics and metadata through various snapshots, each with specific cadences.
+This section explains the types of data collected and when different snapshot types are sent.
+
+There are three different types of snapshots that are sent to the pganalyze app.
+
+* **Compact Activity Snapshot**: Sent every 10 seconds
+* **Compact Logs Snapshot**:
+  * Streaming: Sent approximately every 10 seconds
+  * Download: Sent every 30 seconds
+* **Full Snapshot**: Sent every 10 minutes (logged as just "snapshot")
+
+Successful submissions are logged as follows:
+
+```
+Submitted compact activity snapshot successfully
+Submitted compact logs snapshot successfully
+Submitted snapshot successfully
+```
+
+These snapshots include metrics and metadata collected according to specific schedules and serve the following purposes:
+
+| Data             | Collection Interval | Snapshot Type    | Features                    |
+|------------------|---------------------|------------------|-----------------------------|
+| Connections      | Every 10 seconds    | Compact Activity | Connections                 |
+| Query Stats      | Every 1 minute      | Full             | Query Performance           |
+| Other Stats      | Every 10 minutes    | Full             | Schema Statistics, System, Index Advisor, VACUUM Advisor |
+| Logs (Streaming) | Every 10 seconds    | Compact Logs     | Log Insights, EXPLAIN plans |
```

Comment:
Just realized that I should also set Approx. or something for this one (e.g. "Approx. every 10 seconds")

Reply:
I think if we say "approximately" above, we can leave this as is to conserve some space.